README.md

```diff
@@ -5,6 +5,50 @@ Tools for building Pyodide.
 See [http://github.com/pyodide/pyodide](http://github.com/pyodide/pyodide) for
 more information.
 
+## Quickstart
+
+### Usage
+
+> [!TIP]
+> Currently, installing `pyodide-build` is preferred to running it.
```

Comment:
What does this mean? I usually don't have the question "should I install this or should I run it" because usually I have to install first in order to run?

Reply:
Ah, I meant to convey that users are better off installing `pyodide-build` (globally or in a virtual environment) rather than trying to use `pipx` or `uvx` (instructions for which are provided for completeness). I'll try to reword this sentence.

Reply:
I hope 3ec2ab2c63679c29a82b4fc6f7416bc7c29ad52b conveys the meaning better!